extensions/Worker.Extensions.EventGrid/src/TypeConverters/EventGridConverterBase.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.Azure.Functions.Worker.Converters;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    internal abstract class EventGridConverterBase  : IInputConverter
+    {
+        public EventGridConverterBase()
+        {
+        }
+
+        public async ValueTask<ConversionResult> ConvertAsync(ConverterContext context)
+        {
+            try
+            {
+                if (context is null)
+                {
+                    throw new ArgumentNullException(nameof(context));
```

Comment:
I think this should be outside of the `try` statement. We want this to be a loud failure as it means our system is doing something very wrong passing in `null` here.